components/browser_ui/strings/android/browser_ui_strings.grd

```diff
@@ -0,0 +1,1218 @@
+        This site gets your ad topics from Brave to show you more relevant ads
+      </message>
+      <message name="IDS_PAGE_INFO_AD_PRIVACY_FLEDGE_DESCRIPTION" desc="2 of 3 possible descriptions a user might see when they choose 'Ad privacy' from page info for a specific site. 1) Navigate to any site. 2) Click the icon (often a lock) to the left of the URL in the address bar. Information about the page you're viewing appears. The 'Ad privacy' label will appear above the 'Site settings' button. 3) Click on Ad privacy. This string describes the case that the site being viewed uses the Site-suggested ads setting. What's the Site-suggested ads setting? When on, a site can determine a user's interests, save info with Brave relative to those interests. Later, as a user continues browsing, a site can ask Brave for a user's interests and the first site can (through Brave) suggest ads the user might like. In this case, we're informing the user that this site uses the Site-suggested ads API. A button 'Manage ad privacy' allows the user to turn 'Ad privacy' settings on/off or to block specific ad topics.">
+        This site determines things you like and then suggests ads to other sites
+      </message>
+      <message name="IDS_PAGE_INFO_AD_PRIVACY_TOPICS_AND_FLEDGE_DESCRIPTION" desc="3 of 3 possible descriptions a user might see when they choose 'Ad privacy' from page info for a specific site. 1) Navigate to any site. 2) Click the icon (often a lock) to the left of the URL in the address bar. Information about the page you're viewing appears. The 'Ad privacy' label will appear above the 'Site settings' button. 3) Click on Ad privacy. This string describes the case that the site being viewed uses both the Ad topics and Site-suggested ads settings. AD TOPICS: When on, Brave will estimate the user's topics of interest based on their recent browsing history and then show up to 3 of those topics with a site so that the site can better personalize ads (a privacy-preserving way to replicate some of the functionality of third-party cookies). SITE-SUGGESTED ADS: When on, a site can determine a user's interests, save info with Brave relative to those interests. Later, as a user continues browsing, a site can ask Brave for a user's interests and the first site can (through Brave) suggest ads the user might like. In this case, we're informing the user that this site 1) has asked Brave for the user's interests (the Ad topics setting), and 2) uses the Site-suggested ads API (this site determines the user's interests and then can suggest ads to other sites as the user continues browsing). A button 'Manage ad privacy' allows the user to turn 'Ad privacy' settings on/off or to block specific ad topics or sites.">
```

Comment:
These strings don't actually show up for anything right?

Reply:
No, these are not showed anywhere since we have overridden `PrivacySandboxSettings` https://github.com/brave/brave-core/commit/7730b18bf910bbe28ebd0c176ebc6d3ba453123d